docs/contribution/release-schedule.md

```diff
@@ -0,0 +1,99 @@
+# Spirit Design System Release Schedule
+
+This is a living document outlining the plan for previous, current, and future
+major versions of the Spirit Design System.
+
+| Release      | Status      | Initial release | Begin Active | Begin Maintenance | End of Life |
+| ------------ | ----------- | --------------- | ------------ | ----------------- | ----------- |
+| `main`       | unstable    | unstable        | unstable     | unstable          | unstable    |
+| v0           | End of Life | 2021-09-20      | 2021-09-20   | 2023-07-21        | 2024-06-10  |
+| v1 🌟 Aura   | End of Life | 2023-07-21      | 2023-07-21   | 2024-06-10        | 2024-10-17  |
+| v2 🍃 Breeze | Maintenance | 2024-06-10      | 2024-06-10   | 2024-11-25        | -/-         |
+| v3 ✨ Charm  | Active      | 2024-10-17      | 2024-11-25   | -/-               | -/-         |
+| v4 🌈 Divine | Unreleased  | -/-             | -/-          | -/-               | -/-         |
+
+> Dates are subject to change
+
+![schedule][release-schedule-static]
+
+```
```

Comment:
**question (blocking):** Do we want the schedule in textual view or rather as an image?

Reply:
Image generated by https://github.com/lmc-eu/spirit-design-system/pull/1792

Reply:
OK.